md/release-notes.md

```diff
@@ -42,37 +48,53 @@ For existing artifacts, nothing has changed: their IDs is a UUID. You can switch
+
 ### BDM meta-information for Portal Administrator
 Since installing or updating BDM requires BPM Services to be paused, such rights are only granted to the Technical User.
 However, since the Portal Administrator can now install and update the BDM Access Control file from the Administrator Portal, he can now get information on BDM status and last update date.
 
 ### New layout for the Administrator Portal "User details" page
 All details of a user in the organization are now diplayed on an AngularJS page of the Administrator Portal, bringing still more consistency to the Administrator Portal.
 
+### Bonita Studio status bar and coolbar
+To allow you to work in Bonita Studio while some work is done by Bonita Engine in the background, the Eclipse status bar is now displayed at the bottom right hand corner of the studio window. 
+To keep the same working space in the studio, we have set the default size of the coolbar at "small". If you prefer big icons, with labels, go to "Edit > Preferences > Appearance" and set the "Coolbar default size" to "Normal".
+
 ### Business objects relationships
 The default relationship type between business objects is now "agreggation". It was "composition" before, which led to some validation errors: a single object cannot be linked to several parents with a composition relationship, but it can with an aggregation relationship.
 
-### API error messages
-Improvements of messages in case of error on API calls have been done: explain the cause and give options for easier error recoveries.
-
 ### Studio status bar
 At the bottom right hand corner of Bonita Studio, the Eclipse status bas has been added to give information about the back-end activities, especially in case of operations on remote repositories. In order not to lose diagram designing space, icons of the coolbar are now small by default. If you prefer them large, go to the studio Preferences menu.
 
 ### Application descriptor "deploy"
-Just like for profiles or BDM access control, there is now a "Deploy" option in the Application descriptor menu of Bonita Studio.
+Just like for profiles or BDM access control, there is now a "Deploy" option in the Application descriptor menu of Bonita Studio, on top of the toolbar button already available in the editor.
+
+### Application descriptor refactoring upon a change on a profile
+When a change is made on a profile in Bonita Studio, the application descriptors that embed the profile are now refactored in Bonita Studio to take the change into account.
+
+### French repository is now "dépôt"
+For local or shared repositories, Bonita Studio in French is not tuned to the official translation: "dépôt".
 
 ### Official signatures on Windows
 Bonita binaries as well as Studio installer are now officially signed on Windows OS, to prevent useless security alerts.
 
+### API error messages
+Improvements of messages in case of error on API calls have been done: explain the cause and give options for easier error recoveries.
```

Comment:
What kind of APIs are we talking about?

Reply:
I'm remonving this item. I don't remember, I can't find the Jira issue, and I think this is ok not to mention this is in the release note